src/components/results/securityanalysis/security-analysis-result-tab.tsx

```diff
@@ -103,10 +176,26 @@ export const SecurityAnalysisResultTab: FunctionComponent<
     };
 
     const handleTabChange = (event: SyntheticEvent, newTabIndex: number) => {
-        setResult(null);
+        resetResultStates();
         setTabIndex(newTabIndex);
     };
 
+    // Pagination, sort and filter
+    const handleChangePage = useCallback(
+        (_: any, newPage: number) => {
```

Comment:
the right signature is (event: React.MouseEvent | null, page: number),  avoid using any and rename newPage to page or selectedPage, i think it's more clear

Reply:
I think that is better to use 
React.MouseEvent<HTMLButtonElement> | null

for the type (source, Mui TablePagination component)

then I think that selectedPage is better.